hms-app/src/components/card-details/IdeaDetails.tsx

```diff
@@ -339,6 +339,32 @@ export default function IdeaDetails(props: IProps) {
     )
   }
 
+  const skillsRequared = (
```

Comment:
you probably want to name it `skillsRequired` or `requiredSkills`

Reply:
renamed